packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+## Guidelines
+
+### General
+
+-   **Use links**, instead, to **redirect users to another page** of the current website or app, or to another website.
```

Comment:
Suggestion:
To **redirect users to another page** of the current website or app, or to another website, **use links** rather than buttons.

Reply:
There is a lot of bolding going on here, and I was wondering what is the idea behind it?
(I don't want to upset your practices, I'm mainly just curious because we're very parsimonious with bolding in the doc team generally :) )

Reply:
The principle that I like to use when using bold text is to help readers read faster by highlighting the important words. This way, readers can only read the bold text and understand the basic ideas behind the text.

TLDR style